src/api/libcellml/units.h

```diff
@@ -400,6 +400,33 @@ class LIBCELLML_EXPORT Units: public NamedEntity, public ImportedEntity
      */
     static double scalingFactor(const UnitsPtr &units1, const UnitsPtr &units2);
 
+    /**
+     * @brief Return a bool which indicates if the units are loosely comparable.
+     * 
+     * Two units are loosely comparable if they share the same units ONLY (kg,s,m,...),
+     * this means we are uninterested in unit multipliers. 
+     *
+     * @param1 units1 The first units to compare.
+     * @param2 units2 The second units to compare.
+     *
+     * @return bool indicating whether units are comparable (true), false otherwise.
+     */
+    static bool isEquivalentTo(const UnitsPtr &units1, const UnitsPtr &units2);
+
+    /**
+     * @brief Return a bool which indicates if the units are directly comparable.
+     * 
+     * Two units are dimensionally comparable if they are the same unit, e.g. volts and
+     * volts. This means we can check for the same base units across all parameters (kg,s,m), 
+     * using the exponent, multiplier, and prefix of the unit.
+     * 
+     * @param1 units1 The first units to compare.
+     * @param2 units2 The second units to compare.
+     *
+     * @return bool indicating whether units are dimensionally comparable (true), false otherwise.
+     */
+    static bool isDimensionallyEquivalentTo(const UnitsPtr &units1, const UnitsPtr &units2);
```

Comment:
as above, I'd go with `areDimensionallyEquivalent` or `dimensionallyEquivalent`.

Reply:
I would go with `dimensionallyEquivalent`.

Reply:
@agarny did you make those changes in the commits? (Just checking as I don't want to re-commit changes which have been already been made).

Reply:
Otherwise I'll sort those requested changes now.

Reply:
Commits e30945b and a16e090 are indeed your branch. Just do a `git pull`.